src/components/dashboard/GapByRace.vue

```diff
@@ -58,37 +145,67 @@ const activeStats = computed(() => {
   );
   if (row) {
     // don't let a population be more than 100% vaccinated
-    return fieldData.value.map((f) => {
-      const population = row[f.populationField];
-      return {
-        name: f.name,
-        pct:
-          population > 0
-            ? Math.min(1, row[f.observedField] / row[f.populationField])
-            : 0,
-        gap: Math.max(0, row[f.expectedField] - row[f.observedField]),
-        population,
-      };
-    });
+    return fieldData.value
+      .map((f) => {
+        const population = row[f.populationField];
+        return {
+          name: f.name,
+          pct:
+            population > 0
+              ? Math.min(1, row[f.observedField] / row[f.populationField])
+              : NaN,
+          gap: Math.max(0, row[f.expectedField] - row[f.observedField]),
+          population,
+        };
+      })
+      .sort(sortByProperty("pct"));
   } else {
     return [];
   }
 });
 
+// activeStats is sorted by pct, minVaxRace is the first group whose percentage gap is not NaN
 const minVaxRace = computed(() => {
-  let minRow = activeStats.value[0];
-  activeStats.value.forEach((stat) => {
-    if (stat.pct < minRow.pct) {
-      minRow = stat;
+  for (let i = 0; i < activeStats.value.length; i++) {
+    if (!isNaN(activeStats.value[i].pct)) {
+      return activeStats.value[i];
     }
-  });
-  return minRow;
+  }
```

Comment:
I'd swap this out for a more modern js loop style (does the same exact thing though)
```js
for (const activeStat of activeStats.value) {
  if (!isNaN(activeStat.pct)) {
    return activeStat;
  }
}
```

Reply:
Ahh I was psyching myself out which for/or or for/each ran iteratively or in parallel. Good to know that for/or is iterative! Updated !